src/localization.js

```diff
@@ -287,6 +310,11 @@ angular.module('ngLocalize')
             getKey: getKey,
             setLocale: setLocale,
             getLocale: getLocale,
-            getString: getLocalizedString
+            getString: getLocalizedString,
+            $$bundleReady: function (path) {
```

Comment:
don't need this anymore imo as you can listen in on `localeEvents.resourceUpdates`